pkg/pod/pod.go

```diff
@@ -724,6 +724,25 @@ func (builder *Builder) WithHostNetwork() *Builder {
 	return builder
 }
 
+// WithHostPid allows a pod to have access to the host process ID namespace.
+func (builder *Builder) WithHostPid() *Builder {
```

Comment:
Add boolean flag as a parameter. May be in some cases you'll need explicitly false flag.